composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php": "^7.2.5",
+        "php": "^7.2.5 || ^8",
```

Comment:
Can you use `>=7.2.5` instead? That's what we decided to use on my other projects.

Reply:
Done!